ghost/mariadb/configmap-mariadb.yaml

```diff
@@ -7,32 +7,26 @@ metadata:
   labels:
     app.kubernetes.io/name: mariadb
 data:
-  my.ini: |-
-    [mysqld]
-    skip-name-resolve
-    explicit_defaults_for_timestamp
-    basedir=/opt/bitnami/mariadb
-    plugin_dir=/opt/bitnami/mariadb/plugin
-    port=3306
-    socket=/opt/bitnami/mariadb/tmp/mysql.sock
-    tmpdir=/opt/bitnami/mariadb/tmp
-    max_allowed_packet=16M
-    bind-address=*
-    pid-file=/opt/bitnami/mariadb/tmp/mysqld.pid
-    log-error=/opt/bitnami/mariadb/logs/mysqld.log
-    character-set-server=UTF8
-    collation-server=utf8_general_ci
-    slow_query_log=0
-    slow_query_log_file=/opt/bitnami/mariadb/logs/mysqld.log
-    long_query_time=10.0
-
-    [client]
-    port=3306
-    socket=/opt/bitnami/mariadb/tmp/mysql.sock
-    default-character-set=UTF8
-    plugin_dir=/opt/bitnami/mariadb/plugin
-
-    [manager]
-    port=3306
-    socket=/opt/bitnami/mariadb/tmp/mysql.sock
-    pid-file=/opt/bitnami/mariadb/tmp/mysqld.pid
+  cm.mysqld.skip-name-resolve: "false"
```

Comment:
I assume there's also a program to generate this format from INI?

Reply:
Well, I actually just use the Sublime multi-line operation to convert the file. 
To better reflect the [Option 2 approach](https://github.com/GoogleContainerTools/kpt/issues/3210#issuecomment-1198791600), I added the python script to generate this from INI.